### ISSUE TITLE
Update alt text description in read aloud fundamental title 

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/reading-system-testing#14]
The test is OK, but the text surrounding it should be more clear that alt text speaking must be able to turned on or off. The tests that have been reported are fine because this is how we have been reporting.
